test/releasers/node.ts

```diff
@@ -190,5 +190,16 @@ describe('Node', () => {
       await releasePR.run();
       req.done();
     });
+    it('does not support snapshot releases', async () => {
+      const releasePR = new Node({
+        repoUrl: 'googleapis/node-test-repo',
+        releaseType: 'node',
+        // not actually used by this type of repo.
+        packageName: 'node-test-repo',
+        apiUrl: 'https://api.github.com',
+        snapshot: true,
+      });
+      await releasePR.run();
```

Comment:
This test doesn't assert anything - does it only fail via nock disabling network access?

Reply:
Yes, we haven't mocked any of the routes, so it fails loudly with a ton of attempted network connections.